src/test/java/tech/jhipster/lite/generator/server/springboot/thymeleaf/template/domain/ThymeleafTemplateModuleFactoryTest.java

```diff
@@ -18,14 +19,146 @@ class ThymeleafTemplateModuleFactoryTest {
+        )
+        .containing(
+          "\"build:js\": \"path-exists src/main/resources/static/js && (mkdirp target/classes/static/js && babel src/main/resources/static/js/ --out-dir target/classes/static/js/) || echo \\\"No 'src/main/resources/static/js' directory found.\\\"\""
+        )
+        .containing(
+          "\"build:svg\": \"path-exists src/main/resources/static/svg && recursive-copy \\\"src/main/resources/static/svg\\\" target/classes/static/svg -w -f \\\"**/*.svg\\\" || echo \\\"No 'src/main/resources/static/svg' directory found.\\\"\""
+        )
+        .containing("\"build-prod\": \"NODE_ENV='production' npm-run-all --parallel build-prod:*\"")
+        .containing("\"build-prod:html\": \"npm run build:html\"")
+        .containing("\"build-prod:css\": \"npm run build:css\"")
+        .containing(
+          "\"build-prod:js\": \"path-exists src/main/resources/static/js && (mkdirp target/classes/static/js && babel src/main/resources/static/js/ --minified --out-dir target/classes/static/js/) || echo \\\"No 'src/main/resources/static/js' directory found.\\\"\""
+        )
+        .containing("\"build-prod:svg\": \"npm run build:svg\"")
+        .containing("\"watch\": \"npm-run-all --parallel watch:*\"")
+        .containing("\"watch:html\": \"onchange \\\"src/main/resources/templates/**/*.html\\\" -- npm run build:html\"")
+        .containing("\"watch:css\": \"onchange \\\"src/main/resources/static/css/**/*.css\\\" -- npm run build:css\"")
+        .containing("\"watch:js\": \"onchange \\\"src/main/resources/static/js/**/*.js\\\" -- npm run build:js\"")
+        .containing("\"watch:svg\": \"onchange \\\"src/main/resources/static/svg/**/*.svg\\\" -- npm run build:svg\"")
+        .containing(
+          "\"watch:serve\": \"browser-sync start --proxy localhost:8080 --files \\\"target/classes/templates\\\" \\\"target/classes/static\\\"\""
+        )
+        .and()
       .hasFiles("src/main/resources/templates/index.html")
       .hasFiles("src/main/resources/templates/layout/main.html")
-      .hasFiles("src/main/resources/static/css/application.css");
+      .hasFiles("src/main/resources/static/css/application.css")
+      .hasFiles("postcss.config.js");
+    //@formatter:on
+  }
+
+  @Test
+  void shouldCreateTailwindcssModule() {
+    JHipsterModuleProperties properties = JHipsterModulesFixture
+      .propertiesBuilder(TestFileUtils.tmpDirForTest())
+      .projectBaseName("jhipster")
+      .build();
+
+    JHipsterModule module = factory.buildTailwindcssModule(properties);
+
+    //@formatter:off
+    assertThatModuleWithFiles(
+        module,
+        packageJsonFile(),
+        appPostcssFile(),
+        appCssFile(),
+        appIndexFile()
+    )
+      .hasFile("package.json")
+      .containing(nodeDependency("tailwindcss"))
+      .containing(
+        "\"watch:html\": \"onchange \\\"src/main/resources/templates/**/*.html\\\" -- npm-run-all --serial build:css build:html\""
```

Comment:
it might be clearer with a text block (using three double-quote marks):
```suggestion
        """
        "watch:html": "onchange \"src/main/resources/templates/**/*.html\" -- npm-run-all --serial build:css build:html"
        """
```

Reply:
@murdos: Thank you for the detailed review! The code is now much more readable!